### ISSUE TITLE
rplot doesnt plot data as ordered by corrr

### ISSUE THREAD

**Issue by jamesMo84** [tidymodels/corrr#109]
Hi,
Corrr correlates, rearranges and shapes my data, however when i use rplot, to plot the data following these functions, it displays all data, out of order and not shaped to remove a corner of matrix. Unsure of cause, running R 4.0 and Rstudio 1.3.1056

See below results when running code from CRAN:
x <- correlate(mtcars)
x <- rearrange(x, absolute = FALSE)
x <- shave(x)
x
# A tibble: 11 x 12
   rowname    mpg     vs    drat      am   gear   qsec   carb     hp     wt   disp   cyl
   <chr>    <dbl>  <dbl>   <dbl>   <dbl>  <dbl>  <dbl>  <dbl>  <dbl>  <dbl>  <dbl> <dbl>
 1 mpg     NA     NA     NA      NA      NA     NA     NA     NA     NA     NA        NA
 2 vs       0.664 NA     NA      NA      NA     NA     NA     NA     NA     NA        NA
 3 drat     0.681  0.440 NA      NA      NA     NA     NA     NA     NA     NA        NA
 4 am       0.600  0.168  0.713  NA      NA     NA     NA     NA     NA     NA        NA
 5 gear     0.480  0.206  0.700   0.794  NA     NA     NA     NA     NA     NA        NA
 6 qsec     0.419  0.745  0.0912 -0.230  -0.213 NA     NA     NA     NA     NA        NA
 7 carb    -0.551 -0.570 -0.0908  0.0575  0.274 -0.656 NA     NA     NA     NA        NA
 8 hp      -0.776 -0.723 -0.449  -0.243  -0.126 -0.708  0.750 NA     NA     NA        NA
 9 wt      -0.868 -0.555 -0.712  -0.692  -0.583 -0.175  0.428  0.659 NA     NA        NA
10 disp    -0.848 -0.710 -0.710  -0.591  -0.556 -0.434  0.395  0.791  0.888 NA        NA
11 cyl     -0.852 -0.811 -0.700  -0.523  -0.493 -0.591  0.527  0.832  0.782  0.902    NA

rplot(x)

![image](https://user-images.githubusercontent.com/66101861/87629745-f04f0400-c776-11ea-9352-ed8a552eb246.png)

**Comment by thisisdaryn** [comment]
Hi,

This should be fixed by #99 

Try the development version:
```
# install.packages("remotes") 
remotes::install_github("tidymodels/corrr")
```

**Comment by juliasilge** [comment]
Let us know if you still have trouble after installing the updated version from GitHub!
